services/tables/src/main/java/com/linkedin/openhouse/tables/toggle/repository/BaseToggleStatusesRepository.java

```diff
@@ -3,11 +3,13 @@
 import com.linkedin.openhouse.tables.toggle.model.TableToggleStatus;
 import com.linkedin.openhouse.tables.toggle.model.ToggleStatusKey;
 import java.util.Optional;
+import org.springframework.stereotype.Component;
 
 /**
  * THIS IS A TEMPORARY PLACEHOLDER, without this the whole springboot application is failed to start
  * given missing injection entity for {@link ToggleStatusesRepository}.
  */
+@Component
 public class BaseToggleStatusesRepository implements ToggleStatusesRepository {
   @Override
   public <S extends TableToggleStatus> S save(S entity) {
```

Comment:
findById is not implemented yet?

Reply:
I actually don't find `findById` useful, thus not implementing.

Reply:
whats the purpose of this class then? how does the ToggleStatus get retrieved on Tables?

Reply:
oh .. I thought this is a repository within `housetable`, my bad. 
This `findById` will need to be implemented and is part of the next PR as I planned.  
This PR is  intentionally focusing on HTS only. The only reason that `services/tables/src/main/java/com/linkedin/openhouse/tables/toggle/repository/BaseToggleStatusesRepository.java` showing up in the change list is just for the `component` annotation, otherwise the spring application cannot boot.